### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by matthewTheWizzard**
an English version on the file

### CODE REVIEW

File: react/modules/module04/README.md [hunk 1]
```diff
@@ -1,47 +1,45 @@
...
+What must be done:
 
 
 
-1) Создать отдельную ветку для этого задания
-2) Переписать основную страницу на хуки сохранив функционал. Все тесты по итогу должны работать
-3) Переписать форму на React Hook Form, сохранив функционал. Все тесты по итогу должны работать
+1) Create a separate branch for this task
+2) Saving functionality, rewrite the main page with the hooks. All the test should be working in the end
+3) Saving functionality, rewrite the form with React Hook Form. All the test should be working in the end
```
Comment (lyolya95) [inline react/modules/module04/README.md:20]: it's okay, can we rewrite the last sentences as All tests should be

File: react/modules/module04/README.md [hunk 2]
```diff
@@ -1,47 +1,45 @@
...
 
+### Score
 
-### Оценка
 
+The task will be evaluated by a mentor during the first 6 weeks. To make the evaluation more convenient, create a Pull Request. (DO NOT MERGE).
```
Comment (lyolya95) [inline react/modules/module04/README.md:25]: Also, we can be more polite :)
To make the evaluation more convenient, please, create a Pull Request.

File: react/modules/module04/README.md (hunk 1 above)
Comment (matthewTheWizzard) [reply react/modules/module04/README.md:20]: Is "All the tests should be successfully completed" better?

Comment (lyolya95) [reply react/modules/module04/README.md:20]: Just All tests

**Pull request merged**